core/tests/webdriverio_desktop/explorationTranslationTab.js

```diff
@@ -183,23 +180,6 @@ describe('Exploration translation and voiceover tab', function() {
     await users.logout();
   });
 
-  it('should maintain its active sub-tab on saving draft',
```

Comment:
Is the functionality covered by the deleted tests removed in this PR? That functionality is:
* The translation tab no longer remembers what tab was active when the user publishes changes.
* We no longer have voiceover and translation modes.
* We no longer have translations (this seems unlikely?)

If this functionality is still present, we should have e2e tests for it

Reply:
We have removed the feature to add/edit translations from the exploration editor page, we only do it through the contributor dashboard page now i.e, only translators will be contributing translations! (We will maybe add this feature back in the future if needed!). The adding/removing voiceover feature is still on the exploration editor page!